blog/en/blog/2025/02/28/monthly-report.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Monthly Report (January 27 - February 28)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.api7.ai/uploads/2025/02/28/83pU2BtD_feb-monthly-report-cover-en.png
+---
+
+> We have recently added a new feature within Apache APISIX, which supports `ai-proxy-multi` plugin and `_meta.pre_function` to execute custom logic before execution of each phase. For detailed information, please read the monthly report.
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and is made possible by the collective effort of our community.
+
+From January 27 to February 28, 14 contributors made 39 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.api7.ai/uploads/2025/02/28/JaHy54nf_feb-contributors.png)
+
+![New Contributors List](https://static.api7.ai/uploads/2025/02/28/0jejWHGZ_feb-new-contributors.jpg)
+
+## Recent Feature Highlight
+
+1. [Support `_meta.pre_function` to execute custom logic before execution of each phase](https://github.com/apache/apisix/pull/11793) (Contributor: [Revolyssup](https://github.com/Revolyssup))
+
+Add `pre_function` to the meta of the plugin to allow some personalized code to run before all plugins run. For example, register personalized variables so that these temporary variables can be referenced and used in the plugin.
```

Comment:
```suggestion
Add `pre_function` to `_meta` of the plugin to allow some personalized code to run before all plugins run. For example, register personalized variables so that these temporary variables can be referenced and used in the plugin.
```